packages/hashicorp-vault-signing-manager/src/lib/hashicorp-vault/hashicorp-vault.ts

```diff
@@ -9,6 +10,8 @@ import {
   VaultKey,
 } from './types';
 
+const TIMEOUT = 30 * 1000;
```

Comment:
Should it be configurable?

Reply:
I don't see why someone would really want to change it, requests should be < 1 second. Feels like it would just add to config fatigue exposing it.

If someone really wants it config we can add it in later as a non breaking thing.